README.md

```diff
@@ -94,7 +94,23 @@ Done!
 
 
 ### Installing PhASAR a MacOS system
-Due to unfortunate updates to MacOS and the handling of C++, especially on the newer M1 processors, we can't support native development on Mac.
+To install PhASAR on a Mac you need to follow these steps:
+1. Install both LLVM 14 and LLVM 17 on your system using e.g. homebrew.
+2. Setup LLVM 17 as your compiler to compile PhASAR with using the following environment variables
+
+```
+// this must point to LLVM 17 !
+CC=/opt/homebrew/opt/llvm/bin/clang
+CXX=/opt/homebrew/opt/llvm/bin/clang++
+```
+3. Update your `LDFLAGS`, `CPPFLAGS`, and `PATH` to point to your LLVM 14 installation
+```
+export LDFLAGS="-L/opt/homebrew/opt/llvm@14/lib"
+export CPPFLAGS="-I/opt/homebrew/opt/llvm@14/include"
+export PATH="/opt/homebrew/opt/llvm@14/bin:$PATH"
```

Comment:
Do these flags have to be env variables? Or can they also be supplied to cmake directly?

Reply:
What we could do is define it as a default variable in cmake and users could change it with a cmake flag like
` -DPathToLLVM=/my/custom/llvm/install/llvm@14 `
I think this could make sense.